packages/subgraph/scripts/updateManifest.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-undef */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs/promises');
+const path = require('path');
+
+async function main() {
+  const networks = await fs.readdir('../contracts/deployments', {
+    withFileTypes: true
+  });
+  for (const network of networks) {
```

Comment:
Are you planning to use this script for localhost only?

Reply:
Ohh yes, I've changed the file name to reflect it's intended to be used for localhost only. Let me know if the script itself needs any changes.